modules/addons/realtimeregister_ssl/hooks.php

```diff
@@ -1079,3 +1080,11 @@ function realtimeregister_ssl_overideDisaplayedProductPricingBasedOnConfigOpts($
 
     }
 });
+
+// Check for upgrades
+add_hook('AdminHomepage', 1, function($vars) {
+    require_once __DIR__ . DS . 'Loader.php';
+    new Loader();
+
+    (new Configuration())->upgrade();
```

Comment:
die staat ook in modules/whmcs-addon/modules/addons/realtimeregister_ssl/Addon.php:178 ?